scrapyd_k8s/logging_config.py

```diff
@@ -0,0 +1,29 @@
+import logging
+import sys
+
+VALID_LOG_LEVELS = {
+    'CRITICAL': logging.CRITICAL,
+    'ERROR': logging.ERROR,
+    'WARNING': logging.WARNING,
+    'INFO': logging.INFO,
+    'DEBUG': logging.DEBUG,
+    'NOTSET': logging.NOTSET,
+}
```

Comment:
doesn't Python have something that converts a string to a logging level?

Reply:
logging.getLevelName() - changed the code to use this, it is a bit less convenient but we don't have to store a dict